Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/General/Scripts/Utils/EyeCalibrationChecker.cs

```diff
@@ -16,8 +16,11 @@ public class EyeCalibrationChecker : MonoBehaviour
         [SerializeField]
         private bool editorTestUserIsCalibrated = true;
 
-        public UnityEvent OnEyeCalibrationDetected;
-        public UnityEvent OnNoEyeCalibrationDetected;
+        [SerializeField]
```

Comment:
I don't remember there being code changes as part of these doc updates - did a workspace end up getting overloaded or something?

Reply:
Oh, yeah, I guess that got submitted into the incorrect branch. It was just changing public to SerializedField. I can add it to a different PR if you prefer.